client/core/types.go

```diff
@@ -919,8 +919,7 @@ func token(id []byte) string {
 }
 
 // coinIDString converts a coin ID to a human-readable string. If an error is
-// encountered, the error is logged at Warning level, and the value
-// "<invalid coin>" is returned.
+// encountered the value starting with "<invalid coin>:" prefix is returned.
```

Comment:
Looks like this needed correction since recently.

Reply:
I wish we logged the error here rather than throwing it away...

Reply:
Looks like @buck54321 removed it, if there is no objections I can put it back, I also believe it's best to not loose the original error message when possible
```
log.Warnf("invalid coin ID %x for asset %d -> %s: %v", coinID, assetID, unbip(assetID), err)
```

Reply:
Unfortunate consequence of losing the global logger.
My stance is that fmt.Print is not a sin compared with dropping the error entirely or poisoning the function signatures with a logger argument (terrible IMO).
If we've been fine with dropping it, what's so bad about printing to stdout.